@here/harp-mapview/lib/MapView.ts

```diff
@@ -1839,6 +1844,32 @@ export class MapView extends THREE.EventDispatcher {
         this.m_visibleTiles.markTilesDirty(dataSource);
     }
 
+    /**
+     * Sets the DataSource which contains the elevations, the elevation range source, and the
+     * elevation provider. Only a single elevation source is possible per [[MapView]]
+     *
+     * If the terrain-datasource is merged with this repository, we could internally construct
+     * the [[ElevationRangeSource]] and the [[ElevationProvider]] and access would be granted to
+     * the application when it asks for it, to simplify the API.
+     *
+     * @param elevationSource The datasource containing the terrain tiles.
+     * @param elevationRangeSource Allows access to the elevation min / max per tile.
+     * @param elevationProvider Allows access to the elevation at a given location or a ray
+     *      from the camera.
+     */
+    setElevationSource(
```

Comment:
For uniformity use setElevationProvider() instead?

Reply:
As above, lets discuss in person.